app/Services/Three/PostService.php

```diff
@@ -74,13 +69,10 @@ public function update($signup, $data, $transactionId)
         // Save the new post in Customer.io, via Blink.
         if (config('features.blink') && $postOrSignup instanceof Post && $should_send_to_blink) {
             SendPostToBlink::dispatch($postOrSignup);
-
-            // Log that a post was created.
-            info('post_created', ['id' => $postOrSignup->id, 'signup_id' => $postOrSignup->signup_id]);
         }
 
-        // Add new transaction id to header.
-        request()->headers->set('X-Request-ID', $transactionId);
+        // Log that a post was created.
+        info('post_created', ['id' => $postOrSignup->id, 'signup_id' => $postOrSignup->signup_id]);
```

Comment:
Don't we still need the check that this is a `Post` instance since a signup won't have a `signup_id` property

Reply:
Oh, god. Didn't look closely enough! Fixed in 14e8f51.